platform-sdk/swirlds-platform-gui/build.gradle.kts

```diff
@@ -17,5 +17,4 @@
 plugins {
     id("com.hedera.hashgraph.sdk.conventions")
     id("com.hedera.hashgraph.maven-publish")
-    id("java-test-fixtures")
```

Comment:
This was wrong in #7875. Probably missed due to several rebases.

There are no test fixtures in this project right now, and adding the "java-test-fixtures" will then create an empty Jar, which causes trouble on the Module Path.